distribution/pom.xml

```diff
@@ -27,7 +27,43 @@
         <version>1.3.0-SNAPSHOT</version>
     </parent>
     <artifactId>iotdb-distribution</artifactId>
+    <packaging>pom</packaging>
     <name>IoTDB: Distribution</name>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-server</artifactId>
+            <version>1.3.0-SNAPSHOT</version>
```

Comment:
<version>${project.version}</version> ?

Reply:
Sorry .. shouldn't have replied on my phone ... especially if it resolved the discussion and threw away my text ... so here goes again:

Technically there's no difference between using the one or the other. I actually find it easier to type the real version when adding a new dependency than requiring 3 special characters. But beyond that when switching between versions and branches, I usually find it beneficial to use the real versions as then you don't need to scroll to the top to see which version we're in.

However if it's project consensus to use ${project.version}, I'll change it to that. If it hasn't been discussed, then I'd like to start a discussion to reach consensus on it.

Reply:
And did you stop reviewing after this, or can I interpret it as that this is the only objection you have?

Reply:
If the version could be changed in one mvn command, using real version is better.

Reply:
The maven-release-plugin as well as them maven-versions-plugin perfectly handle this case ... there are issues with some non-standard plugins, that if the version is used in the configuration of there, that they are not found, so I strongly encourage using ${project.version} in plugin configurations, however for dependencies I have never seen these plugins miss anything.

Reply:
And when adding new inner-project dependencies, intelliJ auto-fills it with the current version ... which you manually have to replace with the variable (I should probably add to the list of benefits ;-) )

Reply:
> And did you stop reviewing after this, or can I interpret it as that this is the only objection you have?

Yeah, it is the only question I have.